README.md

```diff
@@ -1 +1,41 @@
-#foo 
+Instructions for running benchmark.py:
+
+
+To start benchmarking, try:
+"python benchmark.py project"
+where project is the name of the json configuration file that you want to run, e.g.
+"python benchmark.py openmdao" to benchmark using "openmdao.json"
+results will be kept in a similarly-named database, e.g. "openmdao.db"
+commands and their output will be kept in a similarly-named log, e.g "openmdao.log"
+
+
+OPTIONS:
+________
+
+To plot a specific spec from the database, after benchmarking has been run, use:
+"python benchmark.py --plot spec"
```

Comment:
you need the project argument here, so it knows what database to pull the data from